install_ddev.sh

```diff
@@ -14,26 +12,110 @@ BINOWNER=$(ls -ld /usr/local/bin | awk '{print $3}')
 USER=$(whoami)
 SHACMD=""
 FILEBASE=""
+CURRENT_DIR=$PWD
+
+#Explain what the script does
+printf "
+${GREEN}
+####
+# This script will install everything you need to participate in this sprint.
+#
+# Feel free to first inspect the script before continuing if you like.
+#  -To do this just open it with a text editor
+#
+# It does the following:
+#  -Install Docker for your OS if you don't have it already
+#  -Install ddev by Drud Technology
+#  -Copy required components to ~/Sites/sprint/
+#  -Do a git pull to ensure you have latest commits for core
+#  -Pre-load docker images for the sprint toolkit:
+#    -Drupal 8
+#    -phpmyadmin
+#    -Coud9 IDE
+#    -Thelounge IRC client
+#
+# Press y to continue
+# !!You don't need to hit enter!!.
```

Comment:
I think that people are going to hit Enter by default. I think this could be a follow-up to either accept Y/N to continue and repeat on any other key.